modules/ossm-rn-fixed-issues.adoc

```diff
@@ -21,6 +21,18 @@ The following issues been resolved in the current release:
 
 * https://issues.redhat.com/browse/OSSM-3025[OSSM-3025] Istiod sometimes fails to become ready. Sometimes, when a mesh contained many member namespaces, the Istiod pod did not become ready due to a deadlock within Istiod. The deadlock is now resolved and the pod now starts as expected.
 
+* https://issues.redhat.com/browse/OSSM-2493[OSSM-2493] Default nodeSelector and tolerations in SMCP not passed to Kiali. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Kiali resource.
+
+* https://issues.redhat.com/browse/OSSM-2492[OSSM-2492] Default tolerations in SMCP not passed to Jaeger. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Jaeger resource.
+
+* https://issues.redhat.com/browse/OSSM-2374[OSSM-2374] Deleting a SMM also deletes the SMMR in OpenShift Service Mesh. The Service Mesh operator no longer deletes the `ServiceMeshMemberRoll` resource when a `ServiceMeshMember` resource is removed manually.
```

Comment:
This needs to be reworded. This bugfix is about an issue where the operator deleted the ServiceMeshMemberRoll when the user deleted one of the ServiceMeshMembers associated with it. While this is the expected behavior when you delete the last ServiceMeshMember, the operator erroneously deleted the ServiceMeshMemberRoll even when the deleted ServiceMeshMember wasn't the last. This is the issue that is now fixed.